satellite/metainfo/metainfo.go

```diff
@@ -506,3 +506,51 @@ func CreatePath(ctx context.Context, projectID uuid.UUID, segmentIndex int64, bu
 	}
 	return storj.JoinPaths(entries...), nil
 }
+
+// checks if bucket has any pointers(entries)
+func (endpoint *Endpoint) checkBucketPointers(ctx context.Context, req *pb.ValueAttributionRequest) (resp bool, err error) {
+	//TODO: Logic of checking if bucket exists will be added in new PR.
+	// write into value attribution DB only if bucket exists but no segments or no bucket and no segments exits
+	defer mon.Task()(&ctx)(&err)
+
+	keyInfo, err := endpoint.validateAuth(ctx, macaroon.Action{
+		Op:            macaroon.ActionList,
+		Bucket:        req.BucketId,
```

Comment:
`Bucket` here is `bucket name` NOT `bucket id`

Reply:
if u look at the metainfoClient
func (metainfo *Metainfo) ValueAttributeInfo(ctx context.Context, **bucket string**, path storj.Path, segmentIndex int64, partnerID string) (err error) {
	defer mon.Task()(&ctx)(&err)

	_, err = metainfo.client.ValueAttributeInfo(ctx, &pb.ValueAttributionRequest{
		PartnerId: []byte(partnerID),
		**BucketId:  []byte(bucket),**
	})

	return err
}

The field is named as **'BucketId"** it is actually the Bucket string typed to []byte and renamed. followed the spec.

Reply:
renamed to bucket name